src/relay/backend/utils.cc

```diff
@@ -368,6 +371,71 @@ void BindParamsInModule(IRModule mod, Map<String, runtime::NDArray> params) {
   BindParamsInModule(mod, params_tmp);
 }
 
+Optional<tir::PrimFunc> DefaultTIRConverterImpl(const Array<te::Tensor>& args,
```

Comment:
Add some comments what this function does.

Reply:
added!